test/jdk/sun/net/InetAddress/nameservice/simple/DefaultCaching.java

```diff
@@ -25,31 +25,20 @@
  * @bug 6442088
  * @summary Change default DNS caching behavior for code not running under
  *          security manager.
- * @run main/othervm/timeout=200 -Dsun.net.inetaddr.ttl=20  DefaultCaching
+ * @run main/othervm/timeout=200 -Djdk.net.hosts.file=DefaultCachingHosts
+ *      -Dsun.net.inetaddr.ttl=20  DefaultCaching
  */
 import java.net.InetAddress;
 import java.net.UnknownHostException;
-import java.security.Security;
 import java.io.FileWriter;
 import java.io.PrintWriter;
 import java.io.BufferedWriter;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
 
 public class DefaultCaching {
 
     public static void main(String args[]) throws Exception {
 
-        String hostsFileNameSrc = System.getProperty("test.src", ".") + "/DefaultCachingHosts";
-        String hostsFileName = System.getProperty("user.dir", ".") + "/DefaultCachingHosts";
-        if (!hostsFileNameSrc.equals(hostsFileName)) {
-            Files.copy(Path.of(hostsFileNameSrc), Path.of(hostsFileName), REPLACE_EXISTING);
-            System.out.println("Host file created: " + hostsFileName);
-        }
-        System.setProperty("jdk.net.hosts.file", hostsFileName);
-        // initial mapping
-        // name service needs to resolve this.
+        String hostsFileName = System.getProperty("jdk.net.hosts.file");
```

Comment:
The same reasoning used in my previous comment can be applied here...

> With the "jdk.net.hosts.file" property specified at runtime, a Hosts File for this test is now instead generated in JTwork/scratch. This does not appear to change the behaviour of the test. However, I would appreciate if someone could verify that this is ok for me. Otherwise I can track back to the original solution.

The test appears to behave the same with the use of a generated Hosts FIle